### PR TITLE
test(card-section-simple): add e2e-storybook test coverage

### LINKED ISSUE #7448: [Test Scenario]: Card section simple with CTA - DDS Consulting
**Engineering info:**  
- HC engineer: Andy Blum
- HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2324
- DDS consulting engineer: @ariellalgilmore 

----

### Component and test scenario title

Card section simple with CTA

### Test scenario steps

a. Check cards, content is loaded and whole card clickable
b. Check heading left aligned.
c. Check the content (heading, copy and arrow link) on the card
d. ~Check the CTA card (color of the card is different)~
e. Customize theme from Carbon theme and verify gray 10, gray 90 and gray 100 theme.

### PR DESCRIPTION
**Pull request by andy-blum**
### Related Ticket(s)

Closes #7448 

### Description

Adds end-to-end storybook tests for the `dds-card-section-simple` web component and `CardSectionSimple` react component

### Changelog

**New**

- Test coverage

### CODE REVIEW

**Comment by andy-blum** [comment]
@ariellalgilmore The ticket calls for a test to "Check the CTA card (color of the card is different)", but that card isn't really special in anyway that's specific to this component, instead, using the `color-scheme` attribute set to `inverse`. Do we actually want to account for that in this test? Or should that be left to tests eventually written for individual cards?

**Comment by ariellalgilmore** [comment]
> @ariellalgilmore The ticket calls for a test to "Check the CTA card (color of the card is different)", but that card isn't really special in anyway that's specific to this component, instead, using the `color-scheme` attribute set to `inverse`. Do we actually want to account for that in this test? Or should that be left to tests eventually written for individual cards?

Agree. I just created a PR for testing the default Card and we are checking the `color-scheme` attribute is set to `inverse` there, so we don't need to check if it works in card-section-simple.

File: packages/react/tests/e2e-storybook/cypress/integration/CardSectionSimple/CardSectionSimple.e2e.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        true
+      );
+    });
+  },
+  checkThemes: () => {
+    cy.carbonThemesScreenshot();
+  },
+};
+
+describe('dds-card-section-simple | default (desktop)', () => {
```
Comment (ariellalgilmore) [inline packages/react/tests/e2e-storybook/cypress/integration/CardSectionSimple/CardSectionSimple.e2e.js:66]: Hi @andy-blum! Tests look good just one small thing for react changing the test names to: 
```suggestion
describe('DDSCardSectionSimple | default (desktop)', () => {
```

**Review by ariellalgilmore — APPROVED** [review]

**Review by jeffchew — APPROVED** [review]
LGTM!

**Pull request merged**